docs/Networking.asciidoc

```diff
@@ -193,6 +193,39 @@ systemctl start SuSEfirewall2
 systemctl enable SuSEfirewall2
 ---------------
 
+*Configure NAT with firewalld*
+
+To configure NAT with firewalld assign bridge interface to internal zone and
+interface with access to the network to external zone:
+```bash
+firewall-cmd --permanent --zone=external --add-interface=eth0
+firewall-cmd --permanent --zone=internal --add-interface=br1
+```
+Reload firewall configuration using `firewall-cmd --reload` command.
+To enable masquerade one can use following command:
+```bash
+firewall-cmd --permanent --zone=external -add-masquerade
+```
+ip_forward is enabled automatically if masquerading is enabled:
+```bash
+cat /proc/sys/net/ipv4/ip_forward
+1
+```
+In case interface is in trusted network it's possible to accept connections by
+default by changing zone target:
+```bash
+firewall-cmd --permanent --zone=external --set-target=ACCEPT
+```
+
+If you don't have firewalld service running, you can use `firewall-cmd-offline`
```

Comment:
If you *do not* have *the* firewalld service running, you can use *the* `firewall-cmd-offline` command for the configuration. Enable *the* service to run on startup: